### PR TITLE
Weather app - Jennifer

### PR DESCRIPTION
**Pull request by Jenniferot**
Link to my deployed project:
https://stupefied-jepsen-5b8fc0.netlify.com

I learnt a lot this weak too. It was really difficult in the beginning, but I got to understand more and more of how an API works and how to get the data you want out of it.

I struggle sometimes with creating functions and structuring my code in the javascript file. I would like to have feedback on how I can make my code more clear.

One thing I couldn't figure out is why my weekdays turns out fine on the computer but on my cell phone I get Invalid Date. It would be great if I could get an answer to why that happens.

I didn't have time to make it responsive but I managed to reach a few stretch goals. 
I kept some code that I don't use (commented it out) because I was thinking that I might check it out later.

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <link href="https://fonts.googleapis.com/css?family=Roboto|Roboto+Condensed|Roboto+Mono|Roboto+Slab&display=swap"
+    rel="stylesheet">
+  <title>Wheather</title>
+</head>
+
+<body>
+  <section>
+    <section class="todayContainer">
+      <header>
+        <h1 id="rubrik">Stockholm</h1>
```
Comment (ericaglimsholt) [inline code/index.html]: Try to add naming in english!

File: code/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        <h3>CURRENT WEATHER</h3>
+      </header>
+      <section id="todayWeather">
+        <section class="info">
+          <p id="todayTemp"></p>
+          <p id="description">Molnigt</p>
+        </section>
+        <section class="symbol">
+          <p id=currentCondition></p>
```
Comment (ericaglimsholt) [inline code/index.html]:
```suggestion
          <p id="currentCondition"></p>
```

File: code/index.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        </section>
+      </section>
+      <section class="todaytime">
+        <section class="todaysunrise">
+          <p class="label">SUNRISE</p>
+          <p id="sr">06:00</p>
```
Comment (ericaglimsholt) [inline code/index.html]: What does `sr` and `ss` stands for?

File: code/script.js [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+      tempContainer.classList.toggle("ok")
+    }
+
+    // Changing the dom
+    document.getElementById("description").innerHTML = `${json.weather[0].description}`
+    document.getElementById("todayTemp").innerHTML = `${shortTemp}˚`
+    document.getElementById("currentCondition").innerHTML += `<img src="https://openweathermap.org/img/wn/${json.weather[0].icon}@2x.png" alt="" />`
+    document.getElementById("sr").innerHTML = `${sunriseTime}`
+    document.getElementById("ss").innerHTML = `${sunsetTime}`
+    //console.log(`${json.main.temp_min}`)
```
Comment (ericaglimsholt) [inline code/script.js]:
```suggestion
```

File: code/script.js [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    //console.log(`${json.main.temp_max}`)
```
Comment (ericaglimsholt) [inline code/script.js]:
```suggestion
```

File: code/script.js [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+  })
+
+
+fetch("https://api.openweathermap.org/data/2.5/forecast?q=Stockholm,Sweden&units=metric&APPID=8e3b288ccb7e41d96fd2f44603191864")
+  .then((response) => {
+    return response.json()
+  })
+  .then((json) => {
+    const filteredForecast = json.list.filter(item => item.dt_txt.includes('12:00'))
+    //console.log(filteredForecast)
```
Comment (ericaglimsholt) [inline code/script.js]:
```suggestion
```

File: code/script.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+   })
+  //console.log(containerFiveDays)*/
+
+/*filteredForecast.forEach(item => {
+  console.log(item.weather[0].description)
+  console.log(item.main.temp)
+  console.log(item.main.temp)
+  console.log(item.main.temp_min)
+  console.log(item.main.temp_max)
+})
```
Comment (ericaglimsholt) [inline code/script.js]: We can delete the code that we don't use :)

File: code/style.css [hunk 8]
```diff
@@ -0,0 +1,172 @@
+body {
```
Comment (ericaglimsholt) [inline code/style.css]: Nice that you've worked with css in that way that it's responsive without need of using media query!

**Comment by Jenniferot** [comment]
Hi Erica! Thanks for the Code review on my wheather app.  Check out this
screenshot, invalid Date on Chrome om my phone. That was the only problem.

All the best,

Jennifer Thorsen



tis 25 feb. 2020 kl. 21:20 skrev Erica Glimsholt <notifications@github.com>:

**Pull request closed without merging**